### DISCUSSION TITLE
How can I choose the editor used in bulkrename?

### DISCUSSION THREAD

**Discussion by amano-kenji** [sxyazi/yazi#1738] [1. Q&A] [answered]
Although EDITOR is helix, yazi bulkrename uses neovim. Why?

**Comment by amano-kenji** [comment]
I just read the codebase, and bulkrename doesn't use `EDITOR` environment variable. It uses the first opener available for `bulk.txt` with `text/plain` mime type.

I think the documentation for `rename` is outdated.

**Comment by sxyazi** [comment, marked as answer]
No, the doc is correct; Yazi's default text opener is `$EDITOR`:

https://github.com/sxyazi/yazi/blob/064517665dabe283d0c31693afd183e6db8a2326/yazi-config/preset/yazi.toml#L32-L34

Added more info to the doc in https://github.com/yazi-rs/yazi-rs.github.io/commit/8a1e0119d08d164d916809265b2cd996f18410e7
